src/Raven.Server/Documents/ETL/Providers/Elasticsearch/ElasticSearchEtl.cs

```diff
@@ -137,7 +137,7 @@ protected override int LoadInternal(IEnumerable<ElasticSearchIndexWithRecords> r
                     }, (i, s, token) => throw new NotSupportedException("We don't use async bulk method"));
 
                     
-                    var bulkIndexResponse = _client.LowLevel.Bulk<BulkResponse>(indexName, streamHandler, new BulkRequestParameters { Refresh = Refresh.WaitFor });
+                    var bulkIndexResponse = _client.LowLevel.Bulk<BulkResponse>(indexName, streamHandler, new BulkRequestParameters { Refresh = Elasticsearch.Net.Refresh.WaitFor});
```

Comment:
Why this got changed?

Reply:
auto-change that popped out due to the ambiguous namespace name, when the new `Raven.Server.Documents.Refresh` namespace was created